### DISCUSSION TITLE
Clicking a link inside dialog/sheet

### DISCUSSION THREAD

**Discussion by shawnmclean** [shadcn-ui/ui#1275] [Q&A] [answered]
Hi!

I have a next.js link inside of a sheet component (like clicking the github avatar on the top right).

In next.js, it navigates to the page, but the sheet still shows.

Is there a way to close the sheet when the link is clicked or navigated away?

What's the recommended method?

eg. https://estatepixie.vercel.app/dashboard

**Comment by shrix1** [comment, marked as answer]
Try this 
```
<SheetClose>
//link
</SheetClose>
```

**Reply by shawnmclean** [reply]
Thank you very much sir! That works!
